packages/node-image-compute/base.packages

```diff
@@ -45,21 +45,18 @@ cray-system-files=1.13.5-2.4_20220927102737__g81928fa
 cray-system-setup-scripts=1.3.2-2.4_20220907182749__ga5ea373
 cray-systemd-presets=1.6.1-2.4_20220907233542__gf8b34cc
 cray-udev-network-cn=1.6.18-2.4_20221025121011__ga8be192
-spire-agent=0.12.2-2.4_20220907184947__ge83309b
```

Comment:
Are we sure we don't need this? Alan seems to need it for the UAN.

Reply:
I was just testing if someone would notice.... probably. I just noticed that it wasn't removed from the other layers. I made an incorrect assumption that they were all related.

Reply:
There's cray-spire-dracut and spire-agent, in meeting the other day we talked about the dracut one so I removed that. I left spire-agent since it wasn't dracut, and I thought we did need it.. but perhaps we don't.

I thought it renewed the spire token or something, Alan said needed it in the UAN image and you were booting the computes the same way as him - figured you'd still need it.